--- a/config.def.h	2019-10-17 14:39:18.012166536 +0300
+++ b/config.def.h	2019-10-17 14:43:40.193930706 +0300
@@ -4,18 +4,16 @@
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
-static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10" };
-static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const int topbar             = 0;        /* 0 means bottom bar */
+static const char *fonts[]          = { "Iosevka:size=11" };
+static const char dmenufont[]       = "Iosevka:size=11";
+static const char col_gray[]        = "#c5c8c6";
+static const char col_black[]       = "#1d1f21";
+static const char col_cyan[]        = "#198844";
 static const char *colors[][3]      = {
-	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+   /*               fg         bg         border   */
+   [SchemeNorm] = { col_gray,  col_black, col_black },
+   [SchemeSel]  = { col_gray,  col_cyan,  col_cyan  },
 };
 
 /* tagging */
@@ -56,7 +54,7 @@
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_black, "-nf", col_gray, "-sb", col_cyan, "-sf", col_gray, NULL };
 static const char *termcmd[]  = { "st", NULL };
 
 static Key keys[] = {
