--- a/config.def.h	2019-10-17 14:39:18.012166536 +0300
+++ b/config.def.h	2019-10-17 14:51:46.019112351 +0300
@@ -26,15 +26,16 @@
 	 *	WM_CLASS(STRING) = instance, class
 	 *	WM_NAME(STRING) = title
 	 */
-	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	/* class        instance    title       tags mask     isfloating   monitor */
+	{ "Firefox",    NULL,       NULL,       1,            0,           -1 },
+	{ "Telegram",   NULL,       NULL,       1 << 8,       0,           -1 },
 };
 
 /* layout(s) */
 static const float mfact     = 0.55; /* factor of master area size [0.05..0.95] */
 static const int nmaster     = 1;    /* number of clients in master area */
-static const int resizehints = 1;    /* 1 means respect size hints in tiled resizals */
+static const int resizehints = 0;    /* 1 means respect size hints in tiled resizals */
+
 
 static const Layout layouts[] = {
 	/* symbol     arrange function */
