--- a/config.def.h	2019-02-09 13:50:41.000000000 +0200
+++ b/config.def.h	2019-10-17 13:30:59.504977381 +0300
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Iosevka:pixelsize=16:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -82,42 +82,47 @@
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+// Base16 Google Dark dark - simple terminal color setup
+// Seth Wright (http://sethawright.com)
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-};
+        /* Normal colors */
+        "#1d1f21", /*  0: Base 00 - Black   */
+        "#cc342b", /*  1: Base 08 - Red     */
+        "#198844", /*  2: Base 0B - Green   */
+        "#fba922", /*  3: Base 0A - Yellow  */
+        "#3971ed", /*  4: Base 0D - Blue    */
+        "#a36ac7", /*  5: Base 0E - Magenta */
+        "#3971ed", /*  6: Base 0C - Cyan    */
+        "#c5c8c6", /*  7: Base 05 - White   */
+
+        /* Bright colors */
+        "#969896", /*  8: Base 03 - Bright Black */
+        "#cc342b", /*  9: Base 08 - Red          */
+        "#198844", /* 10: Base 0B - Green        */
+        "#fba922", /* 11: Base 0A - Yellow       */
+        "#3971ed", /* 12: Base 0D - Blue         */
+        "#a36ac7", /* 13: Base 0E - Magenta      */
+        "#3971ed", /* 14: Base 0C - Cyan         */
+        "#ffffff", /* 15: Base 05 - Bright White */
+
+        /* A few more colors */
+
+        "#f96a38", /* 16: Base 09 */
+        "#3971ed", /* 17: Base 0F */
+        "#282a2e", /* 18: Base 01 */
+        "#373b41", /* 19: Base 02 */
+        "#b4b7b4", /* 20: Base 04 */
+        "#e0e0e0", /* 21: Base 06 */
 
+        [255] = 0,
 
-/*
- * Default colors (colorname index)
- * foreground, background, cursor, reverse cursor
- */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
+        [256] = "#c5c8c6", /* default fg: Base 05 */
+        [257] = "#1d1f21", /* default bg: Base 00 */
+};
+
+// Foreground, background, cursor, and reversed cursor colors
+unsigned int defaultfg = 256;
+unsigned int defaultbg = 257;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
